services-camel/pom.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<artifactId>cdr</artifactId>
+		<groupId>edu.unc.lib.cdr</groupId>
+		<version>4.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>services-camel</artifactId>
+	<version>1.0</version>
```

Comment:
If eclipse doesn't complain, lets remove this so that it will use the same version as the rest of the CDR project.